src/ex4/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "ex4mondayuorimarom",
+  "version": "1.0.0",
+  "description": "You know what they say about #4: you'll remember it *for*ever (ha)",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "watch": "nodemon",
+    "start:dev": "nodemon --ext ",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "nodemonConfig": {
+    "ignore": [
+      "data/*.json",
+      "*.js"
+    ]
+  },
+  "author": "OriMarom",
+  "license": "ISC",
+  "dependencies": {
+    "axios": "^0.27.2",
+    "body-parser": "^1.20.0",
+    "browserify": "^17.0.0",
+    "compression": "^1.7.4",
+    "express": "^4.18.1",
+    "express-async-errors": "^3.1.1",
+    "express-validator": "^6.14.1",
+    "fs": "^0.0.1-security",
+    "node-fetch": "^3.2.6",
+    "require": "^2.4.20",
+    "rimraf": "^3.0.2",
+    "use-spinner": "^0.2.0",
+    "uuid": "^8.3.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/orimaro123/monday-u-exercises.git#ex4"
+  },
+  "bugs": {
+    "url": "https://github.com/orimaro123/monday-u-exercises/issues"
+  },
+  "homepage": "https://github.com/orimaro123/monday-u-exercises/tree/ex4#readme",
+  "devDependencies": {
+    "connect-livereload": "^0.6.1",
```

Comment:
It is recommended to find one relevant package to help you with a certain need.
here you use both `livereload` and `connect-livereload `which are different packages for the same thing, should pick and use one